sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryParsingErrors.scala

```diff
@@ -392,10 +390,9 @@ object QueryParsingErrors extends QueryErrorsBase {
 
   def unexpectedFormatForResetConfigurationError(ctx: ResetConfigurationContext): Throwable = {
     new ParseException(
-      s"""
-         |Expected format is 'RESET' or 'RESET key'. If you want to include special characters
-         |in key, please use quotes, e.g., RESET `ke y`.
-       """.stripMargin.replaceAll("\n", " "), ctx)
+      s"""Expected format is 'RESET' or 'RESET key'. If you want to include special characters
+         |in key, please use quotes, e.g., RESET `key`.""".stripMargin.replaceAll("\n", " "),
```

Comment:
As I see now this the same string as was used in `unexpectedFormatForSetConfigurationError`.
I think we can introduce a `val` for it (in `QueryParsingErrors`) and reuse it.

Reply:
It's very similar but not the same string. One is about SET and the other about RESET.

Reply:
Oh, my mistake. In this case we can do the same refactoring twice.